scripts/PlayoutControl_CLI.py

```diff
@@ -117,130 +116,176 @@
-        playProcess.playout_volume_set(args_main['value'])
-    elif(args_main['command'] == "volume_system_set"):
-        playProcess.playout_volume_system_set()
-
-    elif(args_main['command'] == "volume_up"):
-        playProcess.playout_volume_up()
-    elif(args_main['command'] == "volume_down"):
-        playProcess.playout_volume_down()
-
-    elif(args_main['command'] == "playout_mute_toggle"):
-        playProcess.playout_mute_toggle()
-    elif(args_main['command'] == "playout_mute_status"):
-        if(playProcess.playout_mute_status()):
-            print("True is mute")
-        else:
-            print("False is not mute")
-
-    elif(args_main['command'] == "sys_volume_max_set"):
-        playProcess.sys_volume_max_set(args_main['value'])
-    elif(args_main['command'] == "sys_volume_max_get"):
-        print(playProcess.sys_volume_max_get())
-
-    elif(args_main['command'] == "sys_volume_min_set"):
-        playProcess.sys_volume_min_set(args_main['value'])
-    elif(args_main['command'] == "sys_volume_min_get"):
-        print(playProcess.sys_volume_min_get())
-
-    elif(args_main['command'] == "sys_volume_system_set"):
-        playProcess.sys_volume_system_set(args_main['value'])
-    elif(args_main['command'] == "sys_volume_system_get"):
-        print(playProcess.sys_volume_system_get())
-
-    elif(args_main['command'] == "sys_idle_shutdown_set"):
-        playProcess.sys_idle_shutdown_set(args_main['value'])
-    elif(args_main['command'] == "sys_idle_shutdown_get"):
-        print(playProcess.sys_idle_shutdown_get())
-
-    elif(args_main['command'] == "sys_volume_step_set"):
-        playProcess.sys_volume_step_set(args_main['value'])
-    elif(args_main['command'] == "sys_volume_step_get"):
-        print(playProcess.sys_volume_step_get())
-
-    elif(args_main['command'] == "playout_stop"):
-        playProcess.playout_stop()
-    elif(args_main['command'] == "playout_next"):
-        playProcess.playout_next()
-    elif(args_main['command'] == "playout_restart"):
-        playProcess.playout_restart()
-    elif(args_main['command'] == "playout_pause_toggle"):
-        playProcess.playout_pause_toggle()
-    elif(args_main['command'] == "playout_pause_force"):
-        if(args_main['value']):
-            wait_seconds = int(args_main['value'])
+#####################################
+# START actual command line interface
+
+def PlayoutControl_cli(args_main):
+    if(args_main['command']):
+        # COMMAND given
+        logger.debug('Try command: ' + args_main['command'])
+        if(args_main['command'].startswith("switch_")):
+            # SWITCH attempt
+            args_func = {}
+            args_func['command'] = args_main['command']
+            args_func['value'] = args_main['value']
+            args_func['dirpath'] = args_main['dirpath']
+            playProcess.sys_switches(args_func)
+            return
+
+        elif(args_main['command'] == "playout_position_save"):
```

Comment:
I would still write `if` instead of `elif`

Reply:
Makes sense